doc/classes/MultiMesh.xml

```diff
@@ -106,6 +106,14 @@
 				Sets the [Transform2D] for a specific instance.
 			</description>
 		</method>
+		<method name="set_instance_transform_teleport">
+			<return type="void" />
+			<param index="0" name="instance" type="int" />
+			<param index="1" name="transform" type="Transform3D" />
+			<description>
+				Sets the [Transform2D] for a specific instance without writing motion vectors.
```

Comment:
```suggestion
				Sets the [Transform2D] for a specific instance without writing motion vectors. Use this when teleporting a mesh from a position to another to avoid ghosting artifacts.
```